CMakeLists.txt

```diff
@@ -1,29 +1,56 @@
-project(pivy_cmake_setup NONE)
-cmake_minimum_required(VERSION 3.5)
+project(pivy)
+cmake_minimum_required(VERSION 3.14)
 
+FIND_PACKAGE(SWIG 4.0.0 REQUIRED)
```

Comment:
I would vote for making all primary cmake key words (aka commands) being lowercase. That way the script does not shout at me :) and it's easier to read, IMHO.